lib/data/gallery_options.dart

```diff
@@ -52,6 +53,7 @@ class GalleryOptions {
   final Locale _locale;
   final double timeDilation;
   final TargetPlatform platform;
+  final bool testMode; // True for integration tests.
```

Comment:
I think the name of this could be something like `isTestMode`, just `testMode` makes it sound like an enum.